NMS-14632: Add alt text to Operation (pt. 1) (#5660)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/service-assurance/polling-packages.adoc

> The IP interface and service pages show which polling package and service configuration are applied for this specific service.

Reviewer: ```suggestion
The IP interface and service pages show which polling package and service configuration are applied for this specific service, and associated statistics.
```
Add this only if you agree with comment below.

=== Examples

* `<pattern><![CDATA[^HTTP-(?<vhost>.*)$]]></pattern>`: matches all services with names starting with `HTTP-` followed by a host name.
If the service is called `HTTP-www.example.com`, the Metadata DSL expression `${pattern:vhost}` will resolve to `www.example.com`.
* `<pattern><![CDATA[^HTTP-(?<vhost>.*?):(?<port>[0-9]+)$]]></pattern>`: matches all services with names starting with `HTTP-` followed by a hostname and a port.
There will be two variables (`${pattern:vhost}` and `${pattern:port}`), which you can use in the poller parameters.

Use the service pattern mechanism whenever there are multiple instances of a service on the same interface.
You can make services identifiable by specifying a distinct service name for each instance, but there is no need to add a poller definition for each service.
Common use cases for such services are HTTP virtual hosts where multiple web applications run on the same web server, or BGP session monitoring where each router has multiple neighbors.

== Manually test services

You can run a test via the Karaf shell to test the {page-component-title} polling services:

[source, console]
ssh -p 8101 admin@localhost

> NOTE: In the shell, you can view the command's help information by using `opennms:poll --help`.

Reviewer: ```suggestion
NOTE: In the shell, you can use `opennms:poll --help` to view the command's help information.
```

The following example runs the ICMP monitor on a specific IP interface:

[source, console]
opennms> opennms:poll -S ICMP -P example1 10.23.42.1

The output is verbose, which lets you debug monitor configurations.

> Important output lines are as follows:

Reviewer: ```suggestion
Important output lines include the following:
```

<6> Polling result for the service polled against the IP address.
<7> Response time.

== Manually test filters

Filters are ubiquitous in the {page-component-title} configurations that implement `<filter>` syntax.
You can use the Karaf shell to verify filters.
For more information, see xref:reference:configuration/filters/filters.adoc[].

.SSH to localhost
[source, console]
ssh -p 8101 admin@localhost

> NOTE: In the shell, you can view the command's help information by using `opennms:filter --help`.

Reviewer: ```suggestion
NOTE: In the shell, you can use `opennms:filter --help` to view the command's help information.
```
